services/patient-manager/app/api/models.py

```diff
@@ -22,12 +24,13 @@ class Config:
         extra = Extra.ignore
         allow_population_by_field_name = True
 
-    sex: str
-    name: str
-    birth_date: str
+    first_name: str
+    last_name: str
+    birth_date: date
+    sex: Gender
```

Comment:
Inconsistent to db? Use gender model in db

Reply:
Done